src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicator.java

```
@@ -183,6 +206,230 @@ public void initAverRestCommunicator() throws Exception {
+						onePushFocus();
+						break;
+					}
+
+					if (splitProperty[1].equals(FocusControl.FAR.getName())) {
+						focusFar();
+					} else if (splitProperty[1].equals(FocusControl.NEAR.getName())) {
+						focusNear();
+					}
+					break;
+				}
+				case EXPOSURE: {
+					Command exposureCommand = Command.getByName(splitProperty[1]);
+					switch (exposureCommand) {
+						case BACKLIGHT: {
+							if (value.equals(SWITCH_STATUS_ON)) {
+								backlightOn();
+							} else if (value.equals(SWITCH_STATUS_OFF)) {
+								backlightOff();
+							}
+							break;
+						}
+						case AE_MODE: {
+							if (value.equals(AEMode.FULL_AUTO.getName())) {
+								aeFullAuto();
+							} else if (value.equals(AEMode.MANUAL.getName())) {
+								aeManual();
+							} else if (value.equals(AEMode.IRIS_PRIORITY.getName())) {
+								aeIrisPriority();
+							} else if (value.equals(AEMode.SHUTTER_PRIORITY.getName())) {
+								aeShutterPriority();
+							}
+							break;
+						}
+						case EXP_COMP_DIRECT: {
+							float exposureValue = Float.parseFloat(value);
+							expCompDirect((int) exposureValue);
+							break;
+						}
+						case GAIN_LIMIT_DIRECT: {
+							float gainLimitLevel = Float.parseFloat(value);
+							gainLimitDirect((int) gainLimitLevel);
+							break;
+						}
+						case GAIN_DIRECT: {
+							float gainLevel = Float.parseFloat(value);
+							gainDirect((int) gainLevel);
+							break;
+						}
+						case IRIS_DIRECT: {
+							float irisLevel = Float.parseFloat(value);
+							irisDirect((int) irisLevel);
+							break;
+						}
+						case SHUTTER_DIRECT: {
+							float shutterSpeed = Float.parseFloat(value);
+							shutterDirect((int) shutterSpeed);
+							break;
+						}
+						case AUTO_SLOW_SHUTTER: {
+							if (value.equals(SWITCH_STATUS_ON)) {
+								slowShutterOn();
+							} else if (value.equals(SWITCH_STATUS_OFF)) {
+								slowShutterOff();
+							}
+							break;
+						}
+						default: {
+							break;
```

The default should throw an exception because the cloud sends the weird command. Please also check the rest.